README.md

```diff
@@ -0,0 +1,196 @@
+![Stream Cli](./assets/logo.png)
```

Comment:
let's use same logo as website

Reply:
we can, but someone seemed to made this logo specifically for the CLI so I thought I'll keep on using this. opinions?